### PR TITLE
Change l2fees.info bar

### PR DESCRIPTION
**Pull request by vitali-l2beat**
Resolves L2B-940

https://linear.app/l2beat/issue/L2B-940/implement-changes-for-l2feesinfo-bar-on-the-main-page

Screenshot 2023-03-06 at 11 28 58
Screenshot 2023-03-06 at 11 29 09

### CODE REVIEW

**Comment by krzkaczor** [comment]
image

Can u improve the mobile view for this component?

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
+  return (
+    <div
+      className="
+      mt-[32px] flex flex-row items-center
+      justify-between rounded-[12px] bg-gradient-to-r
+      from-[#7E41CC99]
```
Comment (antooni) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:11]: avoid inlining colors, add them to tailwind config instead

**Comment by antooni** [comment]
image
Agree with @krzkaczor, maybe the font should be smaller?

**Review by antooni — CHANGES_REQUESTED** [review]
Code looks great, with one minor exception (see the comment). Please update the mobile view.

File: packages/frontend/src/components/otherSites/OtherSites.tsx (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/components/otherSites/OtherSites.tsx:11]: yup, and if there is really similar color then use this one instead

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:5]: rename directory from otherSites to other-sites as this is the convention we use

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div
+      className="
+      mt-[32px] flex flex-row items-center
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:9]: mt-[32px] = mt-8

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      justify-between rounded-[12px] bg-gradient-to-r
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:10]: rounded-[12px] = rounded-3

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      from-[#7E41CC99] 
+      via-[#FF46C099] to-[#EE2C0199] px-[40px] py-[24px]
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:12]: do not use the [40px] if you can use px-10 for example, same for py

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    "
+    >
+      <div className="flex flex-col justify-between text-2xl text-white">
+        <span>Ethereum Layer 1 is expensive.</span>
+        <span>How much does it cost to use Layer 2?</span>
+      </div>
+      <OutLink
+        className="
+        flex items-center 
+        rounded-[8px] bg-white
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:23]: same here about using px

File: packages/frontend/src/components/otherSites/OtherSites.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        from-[#7E41CC]
+        to-[#FF46C0]
+        px-[48px] py-[19px] text-lg dark:bg-gradient-to-r"
```
Comment (torztomasz) [inline packages/frontend/src/components/otherSites/OtherSites.tsx:26]: and here

**Review by torztomasz — CHANGES_REQUESTED** [review]

**Review by michalsidzej — COMMENTED** [review]
Are the light mode colors supposed to look like that? 
Screenshot 2023-03-07 at 07 05 33

File: packages/frontend/src/components/other-sites/OtherSites.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+      to-red-200/40 px-10
+      py-6 md:flex-row
+    "
+    >
+      <div className="text-center text-lg text-white md:text-left">
+        Ethereum Layer 1 is expensive. How much does it cost to use Layer 2? 💸
+      </div>
+      <OutLink
+        className="
+        mt-3 ml-0 min-w-[90px] self-center text-center
```
Comment (michalsidzej) [inline packages/frontend/src/components/other-sites/OtherSites.tsx:21]:
```suggestion
        mt-3 ml-0 min-w-[90px]
```
Correct me if I'm wrong, but it seems to me that those classes are not necessary (we're centering this link by flex already).

Comment (vitali-l2beat) [reply packages/frontend/src/components/other-sites/OtherSites.tsx:21]: Yeah, "self-center" is superfluous, but "text-center" is needed

File: packages/frontend/src/components/other-sites/OtherSites.tsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
+  return (
+    <div
+      className="
+      mt-8 flex flex-col items-center justify-between
```
Comment (torztomasz) [inline packages/frontend/src/components/other-sites/OtherSites.tsx:9]: avoid multilines of classnames, if line is too long then maybe use cx module

Comment (krzkaczor) [reply packages/frontend/src/components/other-sites/OtherSites.tsx:9]: don't we have autoformatting for this? :O

**Review by torztomasz — APPROVED** [review]

**Review by antooni — APPROVED** [review]

**Pull request merged**